docs/src/components/AppCacheBanner/index.css

```diff
@@ -1,6 +1,6 @@
 .banner {
   position: fixed;
-  position: sticky;
+  position: sticky; /* stylelint-disable-line */
```

Comment:
Hummmm ok... Not sure if that make total sense to get this in the "standard" config since this kind of tricks are handy when you work with CSS but I guess it's not a big deal for us... poke @davidtheclark @jeddy3